app/src/main/java/agency/five/codebase/android/movieapp/ui/main/MainScreen.kt

```diff
@@ -36,6 +40,11 @@ fun MainScreen() {
         }
     }
     val showBackIcon = !showBottomBar
+
+    val favoritesViewModel = getViewModel<FavoritesViewModel>()
+    val homeViewModel = getViewModel<HomeViewModel>()
+    val movieDetailsViewModel = getViewModel<MovieDetailsViewModel>()
```

Comment:
Inline this. Also you won't be able to initialize movieDetailsVM this way because it will require parameter once you make necessary changes.